examples/lockable-fungible-token/src/lib.rs

```diff
@@ -70,46 +70,54 @@ impl FunToken {
     /// (`predecessor_id`) who is considered the balance owner to the new `allowance`.
     /// If some amount of tokens is currently locked by the `escrow_account_id` the new allowance is
     /// decreased by the amount of locked tokens.
-    pub fn set_allowance(&mut self, escrow_account_id: AccountId, allowance: String) {
+    #[return_result]
+    pub fn set_allowance(
+        &mut self,
+        escrow_account_id: AccountId,
+        allowance: String,
+    ) -> Result<(), &'static str> {
         let allowance = u128::from_str(&allowance).expect("Failed to parse allowance");
```

Comment:
probably preferable if we switch these `.expect` calls over to early returning an error since our error type is a str anyways, but not a big deal if we leave it as is